src/components/ArticleList.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from "react";
+import { getArticles } from "../api";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles()
+      .then((data) => {
+        setArticles(data);
+      });
+  }, []);
+
```

Comment:
I would like you to add a loading state here and use it in this component. At the moment, the user has no idea whether this data is being fetched/loaded, it just shows as a white section for a couple of seconds which can really put a user off. Users dont mind waiting if you tell them something is happening, but you need to tell them. Think about when we play games, we see a loading screen. If you click a button and a black screen shows up, you worry that maybe something has crashed or gone wrong.